src/main/java/it/pagopa/pn/emd/integration/utils/PnEmdIntegrationCostants.java

```diff
@@ -0,0 +1,5 @@
+package it.pagopa.pn.emd.integration.utils;
+
+public class PnEmdIntegrationCostants {
+    public static final String COURTESY_MESSAGE_CONTENT = "Hai ricevuto una notifica";
```

Comment:
ora che lo vedo cosi fisso mi viene in mente che ci chiederanno di cambiarlo... rendiamolo configurabile please